src/Symfony/Component/Messenger/Transport/AmqpExt/Connection.php

```diff
@@ -35,7 +35,7 @@ class Connection
 
     private $connectionConfiguration;
     private $exchangeConfiguration;
-    private $queueConfiguration;
+    private $queuesConfiguration;
```

Comment:
`queueConfigurations`?

Reply:
or event better `queuesOptions`? as this is called that elsewhere?

Reply:
Good push, I've renamed these "configuration" to "options". Makes more sense and is aligned with using the `OptionResolver` component in a following PR.